frontend/components/DocumentFilters/DocumentFilters.jsx

```diff
@@ -129,8 +155,8 @@ export default class DocumentFilters extends Component {
     Object.assign(newFilters[index], filter)
     this.setState({filters: newFilters})
 
-    if (filter.value) {
-      this.updateUrl()
+    if (filter.value !== undefined) {
```

Comment:
We should change that. `=== undefined` has always worked, but previously you could globally defined `undefined` making it a useless exact compare. I'll go through and resolve this in other places if you like. Just raise a ticket.